src/test/java/net/masterthought/cucumber/json/ResultTest.java

```diff
@@ -77,4 +77,28 @@ public void getErrorMessage_ReturnsErrorMessage() {
                 "\tat ✽.And the account balance should be 90(net/masterthought/example/ATMK.feature:12)\n");
     }
 
+    @Test
+    public void getErrorMessageTitle_ReturnsErrorMessageTitle() {
+
+        // given
+        Result result = features.get(1).getElements()[0].getSteps()[5].getResult();
+
+        // when
+        String errorMessage = result.getErrorMessageTitle();
+
+        // then
+        assertThat(errorMessage).isEqualTo("java.lang.AssertionError: ");
+    }
+
+
+    @Test
+    public void getErrorMessageTitle_ReturnsEmptyStringIfNoErrorMessage() {
+
+        Result result = features.get(1).getElements()[0].getSteps()[8].getResult();
+
+        assertThat(result.getErrorMessageTitle()).isEqualTo("");
```

Comment:
if someone leaves \n as the error message I would expect report also displays it as \n instead of empty string - don't you think this would be better?